### DISCUSSION TITLE
Solubility at Ref-pH intrinsic to drug?

### DISCUSSION THREAD

**Discussion by DonkeySaddle** [Open-Systems-Pharmacology/Forum#1722] [Questions & Problems] [answered]
I was working with the Itraconazole simulation found on GIT and noticed the value for this parameter (the parameter being Itracazonle's Solubility at Ref-pH) changed based on whether the patient was fasted or fed.

Why isn't this parameter intrinsic to drug - I assumed it would remain the same regardless of whether or not the patient has been fed?  

![solubility at ref ph](https://github.com/Open-Systems-Pharmacology/Forum/assets/25123758/cee979ae-d2fa-48f0-8a22-6ca3e6893a2c)

**Comment by StephanSchaller** [comment, marked as answer]
I would argue this is not only compound dependent but may also be dependent on the media composition (in addition to pH) in which the compound dissolves. Formulation excipients can have similar effects: https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7284856/

**Reply by DonkeySaddle** [reply]
Thanks, @StephanSchaller. That makes sense to me, and is what I figured. Maybe it's a mistake in the simulation, or compositions of the capsules/solutions are indeeds different from one another (e.g., "Capsule fasted" is actually a different type of capsule than "Capsule fed"), though I am inclined to think it's the former. 

Anyways, thanks again.

DS
